### PR TITLE
Handle Egypt 13 digits mobile numbers

### PR DESCRIPTION
**Pull request by yellowmint**
Enables handling of Egypt numbers like `+20 12X XXX XXX XX`

### CODE REVIEW

**Comment by glizda** [comment]
@yellowmint, were does this request for supporting such number come from? I'm against manual changes to this validation rules. We base this library on `google/libphonenumber` library and use their `PhoneNumberMetadata.xml` ([see the update to metadata version 8.12.18](https://github.com/surgeventures/ex_phone_number/commit/3ab8d5a1820b93d318647bac514ae925894faa8a)).

I see that they have recently updated to version 8.12.19 (https://github.com/google/libphonenumber/commit/ccdf40b623e4211b640c2cae9e9b1a001ba2e4b3). If it helps us with Egyptian numbers, we can use the same update procedure as before.

**Comment by yellowmint** [comment]
Well, it was requested by Bart https://shedul.atlassian.net/browse/ID-29307. Unfortunately update `PhoneNumberMetadata.xml` to newest form `google/libphonenumber` do not solve this problem :/

**Comment by kamilkowalski** [comment]
You can check my reply here: https://shedul.atlassian.net/browse/ID-29307?focusedCommentId=29634

TL;DR that's not a valid phone number unless proven otherwise, let's close the PR please.
